TeamSnapSDK/SDK/Cache/TSPCache.m

```diff
@@ -22,16 +22,26 @@ +(void)setCacheRootPath:(NSURL *)rootPath {
 
 +(NSURL *)cacheRootPath {
     if (!_rootPath) {
-        NSString *directory = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) lastObject];
+        NSString *directory = [NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES) lastObject];
         _rootPath = [[NSURL fileURLWithPath:directory isDirectory:YES] URLByAppendingPathComponent:@"teamsnap"];
     }
     if (![[self fileManager] fileExistsAtPath:[_rootPath path]]) {
         [[self fileManager] createDirectoryAtPath:[_rootPath path] withIntermediateDirectories:YES attributes:nil error:nil];
+        [self markDirectoryForNoBackup:_rootPath];
```

Comment:
According to that doc you sent out, caches aren't backed up regardless. Only the documents directory requires this flag. I don't think it will hurt having it, but I don't think it's necessary.